rust/processor/src/config.rs

```diff
@@ -21,11 +21,12 @@ pub struct IndexerGrpcProcessorConfig {
     pub starting_version: Option<u64>,
     pub ending_version: Option<u64>,
     pub number_concurrent_processing_tasks: Option<usize>,
+    // pub enable_verbose_logging: Option<bool>,
```

Comment:
add verbose here lol

Reply:
oh i guess we don't need to because it's passed as a param in the `run` function

Reply:
is that the right way to do it @larry-aptos ? i'm not completely sure how server-framework works

Reply:
Don't pass it in the run function lmao. This config struct exists so that run doesn't have to change.

Reply:
hmm this is gonna be hard bc it's a command line argument rather than a config.yaml field.